funnel/models/event.py

```diff
@@ -112,6 +146,43 @@ def attendees_by_event(cls, event):
         return db.session.execute(stmt).fetchall()
     __table_args__ = (db.UniqueConstraint("email", "proposal_space_id"), {})
 
+    @classmethod
+    def make_from_dict(cls, participant_dict, space):
+        return Participant(
+            fullname=participant_dict.get('fullname'),
+            email=participant_dict.get('email'),
+            phone=participant_dict.get('phone'),
+            twitter=format_twitter(participant_dict.get('twitter')),
+            job_title=participant_dict.get('job_title'),
+            company=participant_dict.get('company'),
+            city=participant_dict.get('city'),
+            proposal_space=space
+        )
+
+    @classmethod
+    def sync_from_csv(space, csv_file):
+        """csv_file -> name, email, company, twitter
+        """
+        crew_rows = get_rows_from_csv(csv_file)
+        crew_participants = []
+
+        for r in crew_rows:
+            p = space.participants.filter(Participant.email == r[1]).first()
+            if not p:
+                p = Participant(fullname=r[0], email=r[1], company=r[2], twitter=r[3], proposal_space=space)
+                db.session.add(p)
+                db.session.commit()
+            crew_participants.append(p)
+
+        ticket_type = space.ticket_types.filter(TicketType.name == 'Crew').first()
```

Comment:
You shouldn't be harcoding a `Crew` string this deep into the model. Move it to a higher level.